test/e2e/resize_nodes.go

```diff
@@ -388,6 +396,17 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 	// network traffic is unblocked in a deferred function
 }
 
+// shut down eth0 on node for 180 seconds to simulate temporary network partition
+func inititateTemporaryNetworkPartition(c *client.Client, node *api.Node) {
+	host := getExternalIPforSSH(node)
+	Logf("Turning off network interface on %s", node.Name)
+	partitionCmd := "nohup sh -c 'sleep 10 && sudo ifdown eth0 && sleep 180 && sudo ifup eth0' >/dev/null 2>&1 &"
+	if result, err := SSH(partitionCmd, host, testContext.Provider); result.Code != 0 || err != nil {
+		LogSSHResult(result)
```

Comment:
Can you just use `Expect(issuSSHCommand("nohup sh", testContext.Provider, node)).NotTo(HaveOccurred())` ?